doc/user/content/get-started.md

```diff
@@ -73,51 +54,64 @@ Download and install the [PostgreSQL installer](https://www.postgresql.org/downl
 {{< /tab >}}
 {{< /tabs >}}
 
-### Prepare your environment
+## Step 1. Start a free trial
 
-Before getting started, let's prepare an isolated environment for experimenting that guarantees we won't interfere with other workloads running in your database. As a first step, create a new [cluster](/overview/key-concepts/#clusters) with dedicated physical resources:
+With a free trial, you get 14 days of access to Materialize resources worth up to [4 credits per hour](https://materialize.com/pricing/). This limit should accommodate most trial scenarios. However, if you need additional time or resources, reach out to sales@materialize.com.      
 
-```sql
-CREATE CLUSTER quickstart REPLICAS (small_replica (SIZE = '2xsmall'));
-```
+1. [Sign up for a Materialize trial account](https://materialize.com/register/) using your business email address. 
 
-And make it our active cluster:
+1. Check your inbox for a confirmation email from Materialize and follow the instructions to activate your account.
 
-```sql
-SET cluster = quickstart;
-```
+    In most cases, your confirmation should arrive in 30 minutes or less.
 
-Now that **physical isolation** is out of the way, let's also take care of **logical isolation** and create a new schema to make sure we don't run into [namespacing](/sql/namespaces/) annoyances (like naming collisions):
+## Step 2. Prepare your environment
 
-```sql
-CREATE SCHEMA qck;
+In Materialize, a [cluster](/overview/key-concepts/#clusters) is an **isolated workspace**, similar to a database in PostgreSQL or a virtual warehouse in Snowflake. Within a cluster, you have [replicas](/overview/key-concepts/#cluster-replicas), which are the **physical resources** for doing computational work. Clusters are completely isolated from each other, so replicas can be sized based on the specific task of the cluster, whether that is ingesting data from a source, computing always-up-to-date query results, serving results to clients, or a combination.
```

Comment:
The analogy to databases in Postgres is slightly imprecise, since catalog objects are not scoped to a cluster in Materialize (e.g. views and tables are visible from all clusters). Also, you can query a MV or source maintained by one cluster from another cluster, whereas IIRC (could be wrong) cross-database queries aren't possible in Postgres.

Reply:
Removed the postgres mention here, and changed `workspace` to `environment`.